### PR TITLE
feat(24.04): add dosfstools

### PR DESCRIPTION
**Pull request by Meulengracht**
# Proposed changes

Add dosfstools and a test for it

### Forward porting

Can be proposed for 24.10 if approved

## Checklist

* [x] I have read the [contributing guidelines](
https://github.com/canonical/chisel-releases/blob/main/CONTRIBUTING.md)
* [x] I have tested my changes ([see how](https://github.com/canonical/chisel-releases/blob/main/CONTRIBUTING.md#7-test-your-slices-before-opening-a-pr))
* [x] I have already submitted the [CLA form](
https://ubuntu.com/legal/contributors/agreement)

### CODE REVIEW

**Review by clay-lake — APPROVED** [review]
Looks good. Thanks!

File: slices/dosfstools.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package: dosfstools
+
+essential:
+  - dosfstools_copyright
+
+slices:
+  bins:
+    essential:
+      - libc6_libs
+    contents:
```
Comment (cjdcordeiro) [inline slices/dosfstools.yaml:10]: same here: can we add more test coverage pls :)

**Review by cjdcordeiro — CHANGES_REQUESTED** [review]

File: slices/dosfstools.yaml (hunk 1 above)
Comment (zhijie-yang) [reply slices/dosfstools.yaml:10]: It seems there is no need for tests for the bins other than `/usr/sbin/fatlabel`, `/usr/sbin/fsck.fat` and `/usr/sbin/mkfs.fat`, as the other files in `/usr/sbin/` are just symlinks to these three files.

**Review by zhijie-yang — APPROVED** [review]
LGTM, thanks!

File: slices/dosfstools.yaml (hunk 1 above)
Comment (cjdcordeiro) [reply slices/dosfstools.yaml:10]: Noted

**Review by cjdcordeiro — APPROVED** [review]

**Pull request merged**